### PR TITLE
Topics: Add Sample data

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/kafka.py [hunk 1]
```diff
@@ -108,7 +118,8 @@ def next_record(self) -> Iterable[CreateTopicRequest]:
                     logger.info("Fetching topic schema {}".format(topic_name))
                     topic_schema = self._parse_topic_metadata(topic_name)
                     logger.info("Fetching topic config {}".format(topic_name))
-                    topic_request = CreateTopicRequest(
+                    topic = Topic(
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/kafka.py:121]: Hi @harshach, this change looks like a regression to me. We should be aiming at using `Create` requests for the connectors

Comment (pmbrull) [reply ingestion/src/metadata/ingestion/source/kafka.py:121]: Let's tackle this as part of https://github.com/open-metadata/OpenMetadata/issues/1673

**Review by pmbrull — APPROVED** [review]

File: ingestion/src/metadata/ingestion/source/kafka.py [hunk 2]
```diff
@@ -178,11 +191,51 @@ def _parse_topic_metadata(self, topic: str) -> Optional[Schema]:
...
+            logger.error(sys.exc_info()[2])
+        return None
+
+    def on_assign(self, a_consumer, partitions):
+        # get offset tuple from the first partition
+        new_partitions = []
+        for partition in partitions:
+            last_offset = a_consumer.get_watermark_offsets(partition)
+            offset = last_offset[1]
+            print(offset)
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/kafka.py:227]: print!

Comment (ayush-shah) [reply ingestion/src/metadata/ingestion/source/kafka.py:227]: I will handle the same in #4653

**Pull request merged**